From a06d1f46bbb99e7a578c4217353d2b83f1bf383e Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Tue, 27 Jan 2015 13:44:09 +0530
Subject: [PATCH 4/4] bindings/java: Arguments in a shell script's shebang
 aren't portable

This doesn't work on Linux. Use an environment variable instead.
---
 bindings/java/Makefile.am | 2 +-
 bindings/java/gen_jni.py  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/bindings/java/Makefile.am b/bindings/java/Makefile.am
index 7f67468..643defe 100644
--- a/bindings/java/Makefile.am
+++ b/bindings/java/Makefile.am
@@ -40,7 +40,7 @@ OWR_GIR = Owr-0.1.gir
 endif
 
 owr_jni.c: $(OWR_GIR) $(top_srcdir)/bindings/java/
-	$(top_srcdir)/bindings/java/gen_jni.py \
+	PYTHONDONTWRITEBYTECODE=x $(top_srcdir)/bindings/java/gen_jni.py \
 	--gir=$< \
 	--c-out=$(top_builddir)/bindings/java/owr_jni.c \
 	--j-out=$(top_builddir)/bindings/java/owr
diff --git a/bindings/java/gen_jni.py b/bindings/java/gen_jni.py
index 662538d..3576a07 100755
--- a/bindings/java/gen_jni.py
+++ b/bindings/java/gen_jni.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python -B
+#!/usr/bin/env python
 
 # Copyright (c) 2014, Ericsson AB. All rights reserved.
 #
-- 
2.1.0

